app/src/main/AndroidManifest.xml

```diff
@@ -15,6 +16,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Design.NoActionBar"
+        android:usesCleartextTraffic="true"
```

Comment:
Can you tell us why are you using this here?

Reply:
Because I was having an exception related with Http from Glide. So I did a research and I found this solution and I think that is used to allow us to do a http request to a "insecure" host which not use HTTPS

Reply:
Excellent Juli!

Reply:
Thank you Geor!!